lib/BackendClient.js

```diff
@@ -117,8 +117,10 @@ var BackendClient = (function(){
             "partnerId" : item.partnerId,
             "dvrEnabled" : false,
             "recordStatus" : item.recordStatus,
-            "segmentDurationMilliseconds" : item.segmentDuration
+            "segmentDurationMilliseconds" : item.segmentDuration,
+            "capabilities" : item.capabilities
```

Comment:
didn't we drop that?

Reply:
capabilities exist regardless of our usage.